pkg/systemd/overview-cards/systemInformationCard.jsx

```diff
@@ -81,6 +92,43 @@ export class SystemInfomationCard extends React.Component {
                 });
     }
 
+    getSystemUptime() {
+        cockpit.spawn(["cat", "/proc/uptime"])
+                .then(text => {
+                    let uptime_days = 0;
+                    let uptime_hours = 0;
+                    let uptime_minutes = 0;
+
+                    const match = text.match(/[0-9]*\.[0-9]{2}/);
```

Comment:
Let's please not hardcode the assumption that there are always two fractional digits. It may vary by architecture or change in the future. I'd suggest to simply do `const uptime = parseFloat(text.split(' ')[0]).